python/pyspark/ml/tests/test_ovr.py

```diff
@@ -0,0 +1,135 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import tempfile
+import unittest
+
+import numpy as np
+
+from pyspark.ml.linalg import Vectors
+from pyspark.ml.classification import (
+    LinearSVC,
+    LinearSVCModel,
+    OneVsRest,
+    OneVsRestModel,
+)
+from pyspark.testing.sqlutils import ReusedSQLTestCase
+
+
+class OneVsRestTestsMixin:
+    def test_one_vs_rest(self):
+        spark = self.spark
+        df = (
+            spark.createDataFrame(
+                [
+                    (0, 1.0, Vectors.dense(0.0, 5.0)),
+                    (1, 0.0, Vectors.dense(1.0, 2.0)),
+                    (2, 1.0, Vectors.dense(2.0, 1.0)),
+                    (3, 2.0, Vectors.dense(3.0, 3.0)),
+                ],
+                ["index", "label", "features"],
+            )
+            .coalesce(1)
+            .sortWithinPartitions("index")
+            .select("label", "features")
+        )
+
+        svc = LinearSVC(maxIter=1, regParam=1.0)
+        self.assertEqual(svc.getMaxIter(), 1)
+        self.assertEqual(svc.getRegParam(), 1.0)
+
+        ovr = OneVsRest(classifier=svc, parallelism=1)
+        self.assertEqual(ovr.getParallelism(), 1)
+
+        model = ovr.fit(df)
```

Comment:
how about checking ovr.uid == model.uid?

Reply:
SG

Reply:
actually OVRModel doesn't inherit the uid from the OVR, in existing classic implementation.
so I will just skip the uid check